polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -456,18 +507,6 @@ impl<Config: config::Config> XcmExecutor<Config> {
 		Ok(())
 	}
 
-	/// Refund any unused weight.
-	fn refund_surplus(&mut self) -> Result<(), XcmError> {
```

Comment:
nit:

I think `fn subsume_asset()` and `fn subsume_assets()` can also be removed